src/ai-generator/infrastructure/access-token-repository.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Yoast\WP\SEO\AI_Generator\infrastructure;
+
+use RuntimeException;
+use Yoast\WP\SEO\Helpers\User_Helper;
+
+/**
+ * Class Access_Token_Repository
+ * Handles the storage and retrieval of access tokens for users.
+ */
+class Access_Token_Repository implements Token_Repository_Interface {
```

Comment:
This is nitpicking but since this is infrastructure that uses the User Meta to get this AT you could use that in the class name to show this is the User Meta implementation of the generic `Token_Repository`